### ISSUE TITLE
IP from Reverse Proxy is being written into stats db instead of client IP

### ISSUE THREAD

**Issue by totti4ever** [librespeed/speedtest#313]
The client IP apparently is being correctly feteched (#100) and also echoed, but the db column `ip` always holds the reverse proxy's ip

**Comment by adolfintel** [comment]
Yes, that's how it's telemetry.php is implemented. The output of getIP is also stored alongside it, so you'll have both.

**Comment by totti4ever** [comment]
So that's how you still would and want to do it? Or is it that you cannot afford the time to change it, then I could check and make a PR.

To me it doesn't make any sense to store the reverse proxy's IP in every single column again and again and the ISPinfo column is a mixed text column, which is hard to interpret from a technical point of view.

**Comment by adolfintel** [comment]
It's just a remnant from an older version, I never got around to changing it because it never bothered anyone before. Ideally we could move the IP detection code from getIP.php into a file that's included by both getIP.php and telemetry.php, so that they use the same exact code and report the same value. What do you think?

**Comment by totti4ever** [comment]
Hey, sound reasonable, yes. At least that's what I would make it like in pseudo code. Although the name getIP might be the better for the common code and the additional code for the "proccessed IP" might be better in a file like processIP.php or so?

But I am no native coder, so other ideas might weight more.

**Comment by bierpub** [comment]
This issue has been resolved by #457
